pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -740,7 +740,7 @@ func TestController_Service(t *testing.T) {
 			Address:  "10.0.0.1",
 			Ports: model.PortList{
 				&model.Port{
-					Name:     "test-port",
```

Comment:
I think the agreement was to leave the tests unchanged - to verify that behavior didn't change ? Otherwise as Nate mentioned we'll no longer test what happens if someone calls the port 'test-port'

Reply:
This is the unit test. We need to change this because we change the conversion logic.